### PR TITLE
Validation

### PR DESCRIPTION
**Pull request by sidiksam**
##Form validation
### set form validation using JavaScript

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -14,6 +14,38 @@ function closeMenu() {
...
+
+// form.addEventListener('submit', (e) => {
+//   if (!email.value.match(checker)) {
+//     e.preventDefault();
+//     errorDisplay.style.visibility = 'visible';
+//     errorDisplay.classList.add('error-message');
+//     errorDisplay.textContent = '*Your email address should be all in lowercase';
+//   } else {
+//     errorDisplay.style.visibility = 'hidden';
+//   }
```
Comment (tresorsawasawa) [inline script.js:31]: - [ ] For **[best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)**, kindly make sure you remove all the commented code lines to make your code cleaner and professional.

File: script.js [hunk 2]
```diff
@@ -14,6 +14,38 @@ function closeMenu() {
...
+  e.preventDefault();
+  if (inputEmail.value === inputEmail.value.toLowerCase()) {
+    form.submit();
+  } else {
+    inputEmail.focus();
+    errorMessage.innerHTML = 'Email Can not contain Uppercase Letter!';
+    inputEmail.value = inputEmail.value.toLowerCase();
+    errorMessage.style.height = '10px';
+  }
+});
```
Comment (tresorsawasawa) [inline script.js:47]: - [ ] Kindly mamke sure your validation follows the below behavior as required for the project:
  - If a user submits the contact form with the email field containing uppercase characters an error message should show up.
  - If a user submits the contact form with the email field containing only lowercase characters the form is sent normally.
  **Right now, the user can send his form for both situations (with uppercase characters and lowercase characters)**, make sure you fix that.
- [ ] Also make sure, make sure the error message is showed up close to the "submit" button. I can suggest, you display it between the `Textarea` field and the `Get in Touch` button

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @sidiksam 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️

**Hi @sidiksam,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 

128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c

You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**